scipy/special/_add_newdocs.py

```diff
@@ -792,6 +792,21 @@ def add_newdoc(name, doc):
     function by multiplying the result of ``betainc(a, b, x)`` by
     ``beta(a, b)``.
 
+    ``betainc(a, b, x)`` is treated as a two parameter family of functions
+    of a single variable `x`, rather than as a function of three variables.
+    This impacts only the limiting cases ``a = 0``, ``b = 0``, ``a = inf``,
+    ``b = inf``.
+
+    In general
+
+    .. math::
+
+        \lim_{(a, b) \rightarrow (a_0, b_0)} \mathrm{betainc}(a, b, x)
+
+    is treated as a pointwise limit in ``x``. Thus for example,
+    ``betainc(0, b, 0)`` equals ``0`` for ``b > 0``, although it should be
```

Comment:
```suggestion
    ``betainc(0, b, 0)`` equals ``0`` for ``b > 0``, although it would be
```
Maybe to avoid the appearance of suggesting that the behavior *should* be otherwise?

Reply:
Good point.